Resources/public/js/models/ez-locationmodel.js

```diff
@@ -345,6 +345,64 @@ YUI.add('ez-locationmodel', function (Y) {
             return attrs;
         },
 
+        /**
+         * Returns the REST API sort order based on the sortOrder attribute
+         *
+         * @method _getSortDirection
+         * @protected
+         * @return 'ascending' or 'descending'
+         */
+        _getSortDirection: function () {
+            return this.get('sortOrder') === 'ASC' ? 'ascending' : 'descending';
+        },
+
+        /**
+         * Returns the REST API sort clause identifier based on the sortField
+         * attribute.
+         *
+         * @method _getSortClauseIdentifier
+         * @protected
+         * @return {String}
+         */
+        _getSortClauseIdentifier: function () {
+            switch(this.get('sortField')) {
+                case 'MODIFIED':
+                    return 'DateModified';
+                case 'PUBLISHED':
+                    return 'DatePublished';
+                case 'PATH':
+                    return 'LocationPath'; // Not Implemented server side !?!
+                case 'SECTION':
+                    return 'SectionIdentifier';
+                case 'DEPTH':
+                    return 'LocationDepth';
+                case 'CLASS_IDENTIFIER':
+                    return 'ClassIdentifier'; // Not implemented server side !?!
+                case 'CLASS_NAME':
+                    return 'ClassName'; // Not Implemented server side !?!
+                case 'PRIORITY':
+                    return 'Priority'; // Not Implemented server side !?!
```

Comment:
I think @bdunogier renamed this to `LocationPriority` to clarify what it is (when using php api on this, namespace will reflect it).